packages/gui/src/components/plot/add/PlotAddNFT.tsx

```diff
@@ -53,10 +53,10 @@ const PlotAddNFT = forwardRef((props: Props, ref) => {
       title={
         <Flex gap={1} alignItems="baseline">
           <Box>
-            <Trans>Join a Pool</Trans>
+            <Trans>Make plots portable to pools</Trans>
```

Comment:
Can we change this to read:

```suggestion
            <Trans>Plot to a Plot NFT</Trans>
```